about.html

```diff
@@ -101,6 +119,169 @@
+      .start();
+
+  var link = svg.selectAll(".link")
+      .data(json.links)
+    .enter().append("line")
+      .attr("class", function(d) {return "link" + d.value});
+
+  var node = svg.selectAll(".node")
+      .data(json.nodes)
+    .enter().append("g")
+      .attr("class", "node")
+      .call(force.drag);
+
+  svg.append("clipPath")
+    .attr("id", "clipCircle-contributor")
+    .append("circle")
+      .attr("r", radius * scale["contributor"]);
+
+  //uncomment to enable circle clipping for repo icons
+  /*
+  svg.append("clipPath")
+    .attr("id", "clipCircle-repo")
+    .append("circle")
+      .attr("r", radius * scale["repo"]);
+
+  svg.append("clipPath")
+    .attr("id", "clipCircle-center")
+    .append("circle")
+      .attr("r", radius * scale["center"]);
+  */
+
+  node.append("svg:image")
+      .attr("xlink:href", function(d) {return d.image;})
+      .attr("x", function(d) {return -radius * scale[d.type]})
+      .attr("y", function(d) {return -radius * scale[d.type]})
+      .attr("width", function(d) {return radius * 2 * scale[d.type]})
+      .attr("height", function(d) {return radius * 2 * scale[d.type]})
+      .attr("clip-path", function(d) {return "url(#clipCircle-" + d.type + ")" });
+
+  node.append("text")
+      .attr("dx", 20)
+      .attr("dy", ".35em")
+      .text(function(d) { return (showLabel[d.type]) ? d.name : ""});
+
+  force.on("tick", function() {
+    node.attr("cx", function(d) { return d.x = Math.max(radius, Math.min(width - radius, d.x)); })
+        .attr("cy", function(d) { return d.y = Math.max(radius, Math.min(height - radius, d.y)); });
+
+    link.attr("x1", function(d) { return d.source.x; })
+        .attr("y1", function(d) { return d.source.y; })
+        .attr("x2", function(d) { return d.target.x; })
+        .attr("y2", function(d) { return d.target.y; });
+
+    node.attr("transform", function(d) { return "translate(" + d.x + "," + d.y + ")"; });
+  });
+});
+
+</script>
+        </div>
+    </div>
+</section>
+<section>
+    <div class="about-block steering-council gray">
+        <div class="container">
+            <img src="assets/steering.svg" class="img-responsive">
+            <h2>Steering Council</h2>
+            <p class="supportparagraph">Our team is primarily lead by 10 steering committee members who ultimately make the final decisions.</p>
+            <div class="col-md-12">
+            <div class="col-md-4 material material-1">
+               <p class="council-name">Brian Granger<p>
+                <p class="council-info">Cal Poly, San Luis Obispo<br><a href="https://github.com/ellisonbg" target="_blank"><em>@ellisonbg</em></a> on GitHub</p>
+            </div>
+            <div class="col-md-4 material material-1">
+                <p class="council-name">Damian Avila<p>
+                <p class="council-info">Continuum Analytics<br><a href="https://github.com/damianavila" target="_blank"><em>@damianavila</em></a> on GitHub</p>
+            </div></a>
```

Comment:
css tip: you might be able to use [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/%3Anth-child) with first and second to avoid repeating the classes. But I think it's fine for now.